mechanics/TOC-REVIEW.schedule

```diff
@@ -1,6 +1,12 @@
 #@ title: ToC Working Group Update
 #@ duration: 45m
 
+# To generate reminders (on a mac):
+# awk -F \| '/^[0-9]/ {cmd "/bin/date -j -f %FT%T " substr($1, 0, 19) " +%e\\ %b" | getline date; \
+#    cmd "/bin/date -j -f %FT%T -v-4d " substr($1, 0, 19) " +%e\\ %b" | getline reminder; \
```

Comment:
```suggestion
#    cmd "/bin/date -j -f %FT%T -v-4d " substr($1, 0, 19) " +%b\\ %e" | getline reminder; \
```

Slack needs "Aug 8" instead of "8 Aug" when setting a reminder with `/remind`

Reply:
I'm going to merge this and follow with a patch to change. this :)